internal/venice-common/src/main/java/com/linkedin/venice/utils/Utils.java

```diff
@@ -544,6 +544,8 @@ public static String getUniqueTopicString(String prefix) {
     } else if (pubSubTopicType.equals(PubSubTopicType.VIEW_TOPIC)) {
       return getUniqueString(prefix) + Version.VERSION_SEPARATOR + (version)
           + ChangeCaptureView.CHANGE_CAPTURE_TOPIC_SUFFIX;
+    } else if (pubSubTopicType.equals(PubSubTopicType.UNKNOWN_TOPIC)) {
```

Comment:
Looks like this function is used in tests only... can we move it from here?

Reply:
It is used in test only, how about move it to TestUtils.java?

Reply:
SGTM, thanks!